tests/conftest.py

```diff
@@ -94,9 +94,9 @@ def pytest_addoption(parser: pytest.Parser):
     )
     parser.addoption(
         "--deterministic",
-        action="store_true",
-        default=False,
-        help="Turn on deterministic training.",
+        action="store",
```

Comment:
```suggestion
        choice=["true", "false", None],
```
If value should be one of "true", "false", None, then how about using choice?

Reply:
user can set it to true or false. default value is None. If user does not set deterministic, it will be used the task specific fixed value.

Reply:
yes right. so what user can set to `deterministic` is one of three values, right? I think it's better to prevent user from setting other value.